docs/content/getting-started/common-use-cases.md

```diff
@@ -4,9 +4,8 @@ title: Common Use Cases
 seoTitle: Common Use Cases - Exadel Authoring Kit
 order: 2
 -->
-## Common use cases
 
-#### 1. Auto-generated Touch UI dialog, edit config and component detail for an AEM component
+### 1. Auto-generated Touch UI dialog, edit config and component detail for an AEM component
```

Comment:
we have this situation:
```
<h1>...</h1>
<h3>...</h3>
```
NOTE: need to check and fix it here and in all content files, since it's better to keep headers cascading